lib/bolt/bolt_option_parser.rb

```diff
@@ -653,6 +669,61 @@ module            Manage Bolt project modules
           Learn more about Bolt plugins at https://pup.pt/bolt-plugins.
     HELP
 
+    POLICY_HELP = <<~HELP
+      #{colorize(:cyan, 'Name')}
+          policy
+
+      #{colorize(:cyan, 'Usage')}
+          bolt policy <action> [options]
+
+      #{colorize(:cyan, 'Description')}
+          Apply, create, and show policies.
+
+      #{colorize(:cyan, 'Actions')}
+          apply         Apply a policy to the specified targets
+          new           Create a new policy in the current project
+          show          Show available policy
+    HELP
+
+    # CODEREVIEW: Do we want to have a more detailed description for the command
+    #             explaining what a policy is and how to create one?
+    POLICY_APPLY_HELP = <<~HELP
+      #{colorize(:cyan, 'Name')}
+          policy apply
+
+      #{colorize(:cyan, 'Usage')}
+          bolt policy apply <policy> [options]
+
+      #{colorize(:cyan, 'Description')}
+          Apply a policy to the specified targets.
+    HELP
+
+    # CODEREVIEW: Do we want to have a more detailed description for the command
```

Comment:
Definitely not here. If we do anything it should be adding docs and linking to them here like we do for `bolt task` 
```
Documentation
    Learn more about Bolt tasks at http://pup.pt/bolt-tasks.
```